Projects/App/Sources/Apply/ViewModel/ApplyViewModel.swift

```diff
@@ -44,10 +44,14 @@ final class ApplyViewModel: ApplyViewModelProtocol {
 
     func applyButtonTapped() {
         gifticonService.apply(self.gifticonId)
-            .subscribe { [weak self] _ in
-                self?.showToastView.accept(true)
-            } onFailure: { [weak self] _ in
-                self?.showToastView.accept(false)
+            .subscribe { [weak self] applyResponse in
+                if applyResponse.code == "S001" {
+                    self?.showToastView.accept((true, applyResponse.message, nil))
+                } else if applyResponse.code == "F009" {
+                    self?.showToastView.accept((false, applyResponse.message, nil))
+                }
```

Comment:
```suggestion
                if applyResponse.code == "S001" {
                    self?.showToastView.accept((true, applyResponse.message, nil))
                } else  {
                    self?.showToastView.accept((false, applyResponse.message, nil))
                }
```
"F009"가 아닌 에러에대해서도 토스트를 띄울 수 있도록 처리해줘야할 것 같아요!

Reply:
수정했습니다!